src/test/kotlin/apply/application/CheaterServiceTest.kt

```diff
@@ -31,14 +27,14 @@ internal class CheaterServiceTest {
 
     @Test
     fun `부정 행위자를 추가한다`() {
-        given(cheaterRepository.existsByApplicantId(anyLong())).willReturn(false)
-        given(cheaterRepository.save(any(Cheater::class.java))).willReturn(Cheater(1L))
+        every { cheaterRepository.existsByApplicantId(ofType(Long::class)) } answers { false }
```

Comment:
`any()`로 충분해 보이고 필요하다면 `any<Long>()`을 사용하면 어떨까요?

```suggestion
        every { cheaterRepository.existsByApplicantId(any()) } answers { false }
```

Reply:
any() 로 수정할게요! :)